pkgs/applications/misc/welle-io/default.nix

```diff
@@ -0,0 +1,47 @@
+{ stdenv, buildEnv, fetchFromGitHub, cmake, pkgconfig
+, qtbase, qtcharts, qtmultimedia, qtquickcontrols, qtquickcontrols2
+, faad2, rtl-sdr, libusb, fftwSinglePrec }:
+let
+
+  version = "1.0-rc1";
+
+in stdenv.mkDerivation {
+
+  name = "welle-io-${version}";
+
+  src = fetchFromGitHub {
+    owner = "AlbrechtL";
+    repo = "welle.io";
+    rev = "V${version}";
+    sha256 = "1xi59rmk3rdqqxxxrm2pbllrlsql46vxs95l1pkfx7bp8f7n7rsv";
+  };
+
+  cmakeFlags = [
+    "-DRTLSDR=true"
+  ];
+
+  enableParallelBuilding = true;
+
+  buildInputs = [
+    faad2
+    fftwSinglePrec
+    libusb
+    qtbase
+    qtcharts
+    qtmultimedia
+    qtquickcontrols
+    qtquickcontrols2
+    rtl-sdr
+  ];
+
+  nativeBuildInputs = [ cmake pkgconfig ];
+
```

Comment:
I'd appreciate if attributes were in the familiar order: nativeBuildInputs, buildInputs, cmakeFlags, enableParallelBuilding (which is now the default for cmake, qmake and meson projects).